app/src/main/java/com/simple/checkstyledemo/MainActivity.java

```diff
@@ -16,9 +16,7 @@ protected void onCreate(Bundle savedInstanceState) {
     }
 
     private void test(int a) {
-        if (a > 3){
-            return;
-        }
+        if (a > 3) return;
```

Comment:
**Reporter**: PMD
**Rule**: IfStmtsMustUseBraces
**Severity**: WARN
**File**: app/src/main/java/com/simple/checkstyledemo/MainActivity.java L19

Braces https://pmd.github.io/pmd-5.5.1/pmd-java/rules/java/braces.html#IfStmtsMustUseBraces

*<this is a auto generated comment from violation-comments-lib F7F8ASD8123FSDF>*